Boa.Constrictor/WebDriver/Extensions/AbstractWebLocatorListQuestion.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Boa.Constrictor.Screenplay;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Boa.Constrictor.WebDriver
+{
+    /// <summary>
+    /// Abstract class for any Web questions that return a collection of strings using a Web locator and a property name.
+    /// </summary>
+    public abstract class AbstractWebLocatorListQuestion : AbstractWebLocatorQuestion<IEnumerable<string>>
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="locator">The target Web element's locator.</param>
+        public AbstractWebLocatorListQuestion(IWebLocator locator) : base(locator) { }
+
+        #endregion
+
+        #region Properties
+
+        /// <summary>
+        /// Method that returns a string from a Web element input.
+        /// </summary>
+        protected abstract Func<IWebElement, string> Retrieval { get; }
```

Comment:
Any suggestions for a better name than Retrieval?

Reply:
Please see my comments.